eclair-core/src/test/scala/fr/acinq/eclair/channel/publish/ReplaceableTxPublisherSpec.scala

```diff
@@ -290,14 +290,15 @@ class ReplaceableTxPublisherSpec extends TestKitBaseClass with AnyFunSuiteLike w
     }
   }
 
-  test("funding tx not found, skipping anchor output") {
+  // TODO: this test doesn't work anymore because tweaking the commitInput invalidates sigs of the local commitment
+  ignore("funding tx not found, skipping anchor output") {
```

Comment:
The test setup was quite hacky anyway, I think we can remove it, I don't see a good way of testing that scenario cleanly.

Reply:
Done in ce44416a2.